.github/workflows/master.yml

```diff
@@ -97,6 +97,10 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GORELEASER_TOKEN }}
 
+      - if: ! contains(github.ref, "-rc")
```

Comment:
I do not think this is enough... there are all kinds of formats people use -alpha -beta -myrelease are all valid pre-release releases...

Also, this WF doesn't actually create a release, it just creates a Tag... goreleaser then creates a release... which triggers this flow but gets skipped due to the conditions we have in the bump version job.

You might want to add a completely new workflow that only handles "Published" event... then add a condition for prereleases like this: https://github.community/t/release-prerelease-action-triggers/17275/2... will this work? I don't know what krew actually errors on... the format of the semver release tag?